include/circt/Dialect/Pipeline/PipelinePasses.td

```diff
@@ -25,5 +25,14 @@ def ExplicitRegs : Pass<"pipeline-explicit-regs", "pipeline::PipelineOp"> {
   let constructor = "circt::pipeline::createExplicitRegsPass()";
 }
 
+def ScheduleLinearPipeline : Pass<"pipeline-schedule-linear", "pipeline::PipelineOp"> {
+  let summary = "Schedules a linear pipeline.";
+  let description = [{
+    Schedules a linear pipeline based on operator latencies and resource constriants.
```

Comment:
Typo "constriants". Or maybe leave that out as you don't obey resource constraints yet.